### ISSUE TITLE
`case_runners` without augmentation data or explicit `do_compact_updates=False` raises exception

### ISSUE THREAD

**Issue by rtweeks** [PayTrace/intercom_test#13]
[This code](https://github.com/rtweeks/intercom_test/blob/d9dc3de61b7307800cb1ce2f73dbba52deb27d78/lib/intercom_test/framework.py#L183) should be

```python
if do_compact_updates and self._case_augmenter is not None:
```

**Comment by rtweeks** [comment]
I'm working on a patch for this.

**Comment by rtweeks** [comment]
Fixed in v2.3.0
